automation/jleechanorg_pr_automation/tests/test_workflow_specific_limits.py

```
@@ -220,6 +220,282 @@ def test_count_fixpr_ignores_impostors(self):
+    Bead: REV-h9w
+    """
+
+    def setUp(self):
+        """Set up test environment"""
+        with patch('jleechanorg_pr_automation.jleechanorg_pr_monitor.AutomationSafetyManager'):
+            self.monitor = JleechanorgPRMonitor(automation_username="test-automation-user")
+            self.monitor.safety_manager.fixpr_limit = 10
+
+    def test_rolling_window_includes_23_hours_ago(self):
+        """Comments from 23 hours ago should be counted (within 24h window)."""
+        from datetime import timedelta
```

Ruff (0.14.14)

[warning] `import` should be at the top-level of a file

(PLC0415)